### PR TITLE
"AAAAAAAAAAAAAAAAAA", the cyborg said calmly.

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request

https://github.com/user-attachments/assets/06ffa832-1b68-4a8f-b078-baf6aeeefd9a
## Why It's Good For The Game

dalek-daleks
## Changelog
:cl: PixelShader and grungussuss
sound: cyborgs now have a scream sound
/:cl:

### CODE REVIEW

**Comment by DrAmazing343** [comment]
This is gonna get abused SO HARD LMFAOOOO

Speedmerge.

**Comment by MrMelbert** [comment]
I think it's too high pitched

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -368,6 +369,11 @@
 		return
 	return ..()
 
+/datum/emote/living/scream/get_sound(mob/living/user)
+	. = ..()
+	if(iscyborg(user))
+		return 'sound/mobs/non-humanoids/cyborg/cyborg_scream.ogg' //if you add more sounds for living scream - do the same thing human emotes do for get_sound
```
Comment (MrMelbert) [inline code/modules/mob/living/emote.dm:375]:
```suggestion
	if(iscyborg(user))
		return 'sound/mobs/non-humanoids/cyborg/cyborg_scream.ogg' //if you add more sounds for living scream - do the same thing human emotes do for get_sound
	return ..()
```

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Any chance it could be adjusted to roughly the same pitch as the trailing part of the borg deathsound?

**Comment by MrMelbert** [comment]
Yeah the last half second of that just sounds like a normal dude screaming

**Comment by Sadboysuss** [comment]
> Yeah the last half second of that just sounds like a normal dude screaming 

It's intended

If you think it isn't good , go ahead and close it, I have no attachments to it. Will just have to go back to the drawing board

**Comment by MrMelbert** [comment]
Yeah I'm still not a fan, I'd prefer if the sound was closer to that of the cyborg deathsound

**Pull request closed without merging**